contracts/root/Validator.sol

```diff
@@ -0,0 +1,11 @@
+pragma solidity ^0.4.24;
+
+import { ERC721 } from "openzeppelin-solidity/contracts/token/ERC721/ERC721.sol";
+
+contract Validator is ERC721 {
+
```

Comment:
Why not use as different contract instead of extending to StakeManager?

Reply:
yeah, I was kinda thinking like we can use it as storage, push all properties to the validator contract and in stake manager use it on an abstract level(like primitive data type).